examples/sequence-to-sequence/seq2seq_translator/README.md

```diff
@@ -0,0 +1,268 @@
+# Seq2seq Translator Benchmarks
+
+Here is the comparison between Dynet and PyTorch on the [seq2seq translator example](https://pytorch.org/tutorials/intermediate/seq2seq_translation_tutorial.html).
+
+The data we used is a set of many thousands of English to French translation pairs. Download the data from [here](https://download.pytorch.org/tutorial/data.zip) and extract it to the current directory.
+
+## Usage (Dynet)
+
+The architecture of the dynet model `seq2seq_dynet.py` is the same as that in [PyTorch Example](https://pytorch.org/tutorials/intermediate/seq2seq_translation_tutorial.html). We here implement the attention mechanism in the model.
+
+The architecture of the dynet model is shown as follows. 
+
+```python
```

Comment:
I don't think this should be included in `README.md`, because if we make changes to the actual code they might get out of sync. Let's keep the code in `.py` files.

Reply:
Deleted the architecture part in README.md.